senza/cli.py

```diff
@@ -155,6 +155,7 @@ def convert(self, value, param, ctx):
 
 region_option = click.option('--region', envvar='AWS_DEFAULT_REGION', metavar='AWS_REGION_ID',
                              help='AWS region ID (e.g. eu-west-1)')
+parameter_file_option = click.option('--parameter-file', help='Config file for params', metavar='PATH')
```

Comment:
Could we user a better name for `metavar` like `metavar=SENZA_PARAMS_FILE`? Just to not confuse with `PATH` env var.